src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -231,33 +231,34 @@ export const Checkout: React.FC<Props> = ({
       offerData.person.ssn,
   )
 
-  const startSign = async () => {
+  const startSign = () => {
     if (!canInitiateSign) {
       return
     }
 
     const baseUrl = `${window.location.origin}/${locale}/new-member`
-    const result = await signQuotes({
+    signQuotes({
```

Comment:
Just my opinion, I prefer `async/await` 🤓

Reply:
Haha OK I totally respect that! 😄  The thing with using a `Promise` here was 1) that I could do a destructuring of `data` in a (to me) neat way and 2) that it enabled to easily add a `catch` without wrapping it all in a `try/catch` or having yet _another_ `if` to check if the request failed.

Reply:
I have seen the usage of  the "then" to handle promises in our code but also await. I think we should be consistent
and the await syntax is more modern and a bit cleaner especially if have several steps.

Reply:
I feel that both `Promises` and `async/await` they can both be useful for different cases and I'm not really sure that this is a case where you have to pick one? Although I generally love consistency 😀

Reply:
Just to chip in a third opinion i prefer `pipe` from rxjs ;)

Jokes aside i don't really have an opinion either. If we want to go towards a more functional style code base then maybe we should consider using `then` more (as generally it results in chaining functions rather than assigning tons of variables and hence state) but otherwise i agree `async`/`await` is a bit more readable. For this case imo it was nicer with `async`/`await` to be honest, but also we didn't actually catch if there was any errors. Adding a catch around this with nested ifs would decrease the prettiness quite a bit though, so i'd vote for keeping it with `then`s 

**tldr: keep it with `then` imo**